.babelrc

```diff
@@ -1,4 +1,5 @@
 {
   "presets":  ["es2015", "react"],
+  "plugins": ["transform-object-rest-spread"],
```

Comment:
will this allow us to remove the Object.assign syntax in the reducers, or should we still be doing that to create _new_ objects.